components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -128,7 +128,7 @@
     <!-- The content of the notification, this will be shown to the user when more than two newly supported add-ons are available. %1$s is the app name (in most cases Firefox). -->
     <string name="mozac_feature_addons_supported_checker_notification_content_more_than_two">Add them to %1$s</string>
     <!-- This is the caption for not yet supported screen caption -->
-    <string name="mozac_feature_addons_not_yet_supported_caption">Firefox add-on technology is modernizing. These add-ons use frameworks that are not compatible with Firefox 75 &amp; beyond.</string>
+    <string name="mozac_feature_addons_not_yet_supported_caption">We\'re currently building support for an initial selection of Recommended Extensions.</string>
```

Comment:
Unfortunately, I believe when we update a string we need to use a new string key. @flodolo or @Pike can help us to confirm.

Reply:
We do need a new ID. Also, please don't use escaped single quote, use an actual apostrophe `‘`.

Reply:
@flodolo should we delete all the previous strings?

Reply:
Normally, yes, you should remove a string if it's unused. 

But I'm not exactly sure how this work here (@Pike ?). I know Fenix keeps them around until that specific strings it's in use (i.e. shipping), no clue about a-c

Reply:
I don't know the state of release branches at this time.

Reply:
@pocmo or @st3fan 
What is the expectation on your side at this point? I wasn't involved in the original discussion, and I have no clue what's the current status.

Reply:
I see no problem in removing the old string as soon as it is no longer used.